process/transaction/metaProcess.go

```diff
@@ -88,12 +95,23 @@ func (txProc *metaTxProcessor) ProcessTransaction(tx *transaction.Transaction) e
 
 	switch txType {
 	case process.SCDeployment:
-		return txProc.processSCDeployment(tx, adrSrc)
+		err = txProc.processSCDeployment(tx, adrSrc)
```

Comment:
Why not directly: return txProc.processSCDeployment(tx, adrSrc) ?

Reply:
done